src/components/modal.jsx

```
@@ -34,13 +34,13 @@
-    <div className="modal--carrousel">
+    <div className="flex">
       {images.map((image, index) => (
-        <div key={index} className="modal--image-container">
+        <div key={index} className="w-36 h-36 overflow-hidden mr-1 rounded">
           <img 
             src={image.url} 
             alt="Cat Image" 
-            className='modal--image' 
+            className='w-full h-full object-cover rounded transition-transform duration-300 ease-in-out cursor-pointer hover:scale-110 opacity-80 hover:opacity-100' 
             onClick={swapImage} 
             style={{ backgroundImage: `url(${image.url})` }}/>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)